types/jquery/README.md

```diff
@@ -0,0 +1,53 @@
+### Usage
+
+#### Global
+
+When jQuery is globally available, you can use `jQuery` and `$` directly.
+
+#### Importing (with a global DOM available)
+
+When you want to import jQuery as a module and have a global DOM available (e.g. browser and browser-like environments):
+
+```typescript
+import * as jQuery from 'jquery';
```

Comment:
Actually with `allowSyntheticDefaultImports` set to true `import jQuery from 'jquery'` is valid. https://www.typescriptlang.org/docs/handbook/compiler-options.html